recipes/llvm-core/all/readme.md

```diff
@@ -0,0 +1,9 @@
+## Conan package for [LLVM](https://github.com/llvm/llvm-project)
+
+This is a package for just the core libraries, not the compiler nor any of the **clang**-based tools.
+
+🚧 Currently under development... 🚧
+
+**Copyright (c):** 2020 Alberto Lorenzo Márquez.
+
+**License:** MIT (see `../license.txt`).
```

Comment:
This file isn't needed.

Reply:
My bad. It is a leftover as this recipe comes from a personal project. Thank you for noticing it.